# baima tongling
# topic0: &topic0 "/front/rslidar_points"
# topic1: &topic1 "/front_left/rslidar_points"
# topic2: &topic2 "/front_right/rslidar_points"
# topic3: &topic3 "/back/rslidar_points"

# ruomaoshan
# topic1: &topic1 "/front_left/rslidar_points"
# topic2: &topic2 "/front_right/rslidar_points"
# topic3: &topic3 "/back/rslidar_points"

# chongqing
# topic0: &topic0 "/front_lidar"
# topic1: &topic1 "/front_left_lidar"
# topic2: &topic2 "/front_right_lidar"
# topic3: &topic3 "/back_lidar"

bag0: &bag0 "lidar_front.bag"
bag1: &bag1 "lidar_back.bag"


# lidar_topics: &lidar_topics_all [*topic0, *topic1, *topic2, *topic3]
bag_files: [*bag0, *bag1]

threshold_plane: 0.4

front_lidar_height: 1.2

ground_z_threshold: 1.0

front_pose: [
        0, -1, 0, 0,
        1, 0, 0, 0,
        0, 0, 1, 0,
        0, 0, 0, 1
]

left_pose: [
        1, 0, 0, 0,
        0, 1, 0, 0,
        0, 0, 1, 0,
        0, 0, 0, 1
]

right_pose: [
        1, 0, 0, 0,
        0, 1, 0, 0,
        0, 0, 1, 0,
        0, 0, 0, 1
]

back_pose: [
        1, 0, 0, 0,
        0, 1, 0, 0,
        0, 0, 1, 0,
        0, 0, 0, 1
]


fix_front: true
fix_left: false
fix_right: false
fix_back: false
  




      